### PR TITLE
Update the controlbar for the new layout

### PR DESCRIPTION
**Pull request by egreaves**
### Changes proposed in this pull request:
This PR updates the controlbar with the following changes:
- Remove visual playlist and all supporting code
- Remove the previous button
- Place the next button after the duration
- Delete playlist container css rules from skins

Fixes #
JW7-2518

### CODE REVIEW

**Comment by egreaves** [comment]
test this please

**Comment by johnBartos** [comment]
Player size: smaller 👍

File: src/css/imports/icons.less [hunk 1]
```diff
@@ -135,11 +135,4 @@
     &:before {
```
Comment (johnBartos) [inline src/css/imports/icons.less]: Can we get rid of the prev icon too?

File: src/js/view/controlbar.js [hunk 2]
```diff
@@ -302,23 +280,15 @@ define([
         onPlaylist : function(model, playlist) {
             var display = (playlist.length > 1);
             this.elements.next.toggle(display);
-            this.elements.prev.toggle(display);
-            if(this.elements.playlist) {
-                this.elements.playlist.setup(playlist, model.get('item'));
-            }
         },
-        onPlaylistItem : function(model/*, item*/) {
+        onPlaylistItem : function(/*model, item*/) {
```
Comment (johnBartos) [inline src/js/view/controlbar.js]: I think we should just remove these comments, or at least move them out of the signature

File: src/css/imports/icons.less (hunk 1 above)
Comment (egreaves) [inline src/css/imports/icons.less]: Yep - good catch.

**Pull request merged**